flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/reader/MySqlPipelineRecordEmitter.java

```diff
@@ -71,6 +72,9 @@ public class MySqlPipelineRecordEmitter extends MySqlRecordEmitter<Event> {
     // Used when startup mode is initial
     private Set<TableId> alreadySendCreateTableTables;
 
+    // Used when startup mode is snapshot
+    private boolean shouldEmitAllCtesInSnapshotMode = true;
```

Comment:
`Ctes` means?

Reply:
`Ctes` means CreateTableEvents. How do you think it should be modified to make it clearer?

Reply:
Clear is better than long in my opinion